server/services/actions/hpa_vpa_recommendations.py

```
@@ -0,0 +1,488 @@
+         rec_id, org_id, STATUS_PROPOSED),
+    )
+    row = cursor.fetchone()
+    if not row:
+        return None
+    # user_id is the account whose GitHub credential opened the PR. The clicker
+    # is a different person and may have no GitHub connection at all, so the
+    # close must be able to fall back to the opener.
+    keys = ("repo_full_name", "pr_number", "pr_url", "workload", "environment", "service",
+            "autoscaler", "vcs_provider", "recommendation", "severity_score",
+            "slack_channel_id", "slack_message_ts", "user_id")
+    out = dict(zip(keys, row))
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)